Automattic/wp-calypso
# New Instructions step: initial approach to UI for the new instructions screen

Related to https://github.com/Automattic/wp-calypso/issues/89760

## Proposed Changes

This is a rough mockup for the new instructions screen. This PR does not intend to be the final screen but something to help us iterate on the design of this screen. It's worth mentioning that it is behind the feature flag, so it can be merged as-is anytime.

The idea is to get this merged so we can start hooking up the real behaviour.

The state of the actions is hardcoded via a timer, this behaviour will change sooner or later to reflect the actual state of affairs.

This is what it currently looks like:
<p align="center">

https://github.com/Automattic/wp-calypso/assets/375980/38eb6fd2-03bc-4cf9-8cb9-a0e95b8181b1

</p>

## Testing Instructions



* Apply this diff or go to the calypso live link below
* Go through /start flow
* Once you get to the Goals page run this in your console `sessionStorage.setItem('flags', "migration-flow/remove-processing-step")` and reload the page
* Select the import intent
* Keep going until the instructions screen
* Make sure you don't see the processing screen
* Make sure you see a new step (4.) that conveys a list of pending actions
* The list is still hardcoded so no progress will be seen
* Verify you see an animation as the list gets completed

## Pre-merge Checklist
